incubator/opa/templates/opa-rbac.yaml

```diff
@@ -0,0 +1,38 @@
+# Grant OPA/kube-mgmt read-only access to resources. This lets kube-mgmt
+# replicate resources into OPA so they can be used in policies.
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ include "opa.fullname" . }}-viewer
+roleRef:
+  kind: ClusterRole
+  name: view
```

Comment:
View without resource limits feels broad. I might be overthinking it though.

Reply:
The built-in clusterrole view is limited on what resources it can view.  This is necessary so that policies can import any resources that they need to watch.

In the future we could probably write our own clusterrole that corresponds to the policies, but I think right now that's out of scope.  View should be relatively safe.